core/src/main/java/io/confluent/rest/exceptions/KafkaExceptionMapper.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
```

Comment:
Is this the right license for files in `rest-utils`? Other classes seem to have Apache license.